server/routes/sensor-events.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+function initSensorEvents(socket) {
+    // TODO need to discuss rule eslint about 'use-before-define'
+    function setUpdateInterval() {
+        return setInterval(() => {
+            socket.emit('event', {
+                value: ~~(Math.random() * 100),
```

Comment:
This usage 2 bitwise operators is for round to integer (whole), it like Math.round(), full ex.:
```sh
Math.round(Math.random() * 100)
```
But I prefer ~~, because it look shorter